deps/chakrashim/src/v8string.cc

```diff
@@ -82,98 +82,57 @@ int String::Utf8Length() const {
   return str.length();
 }
 
-template <class CharType>
-static int WriteRaw(
-    JsValueRef ref, CharType* buffer, int start, int length, int options) {
-  if (length == 0) {
-    // bail out if we are required to write no chars
-    return 0;
-  }
-
-  jsrt::StringUtf8 str;
-  if (str.From(ref) != JsNoError) {
-    // error
-    return 0;
+int String::Write(uint16_t *buffer, int start, int length, int options) const {
```

Comment:
Hmm, I am unsure what it does exactly now. Will do some tests to find out later. (Maybe create a bug on this.)